### PR TITLE
feat: add loading pulse effect on country view

### PR DESCRIPTION
**Pull request by ArmanpreetGhotra**
Add pulse animation to the country view in nutrition, fcs and ipc.

### CODE REVIEW

File: src/components/Map/FcsChoropleth.tsx [hunk 1]
```diff
@@ -56,6 +56,23 @@ export default function FcsChoropleth({
...
       />
+      {/* Animated GeoJSON layer for the selected country */}
+      {!regionData && selectedCountryId && (
+        <GeoJSON
+          data={
+            {
+              type: 'FeatureCollection',
+              features: data.features.filter((feature) => feature?.properties?.adm0_id === selectedCountryId),
+            } as FeatureCollection<Geometry, GeoJsonProperties>
+          } // Explicitly type as FeatureCollection
```
Comment (bohdangarchu) [inline src/components/Map/FcsChoropleth.tsx:67]: remove comment

File: src/components/Map/FcsChoropleth.tsx [hunk 2]
```diff
@@ -56,6 +56,23 @@ export default function FcsChoropleth({
...
+          style={{
+            color: 'undefined',
+            fillOpacity: 0.3,
+            fillColor: '#338ef7',
```
Comment (bohdangarchu) [inline src/components/Map/FcsChoropleth.tsx:71]: use tailwind colors with `hsl(var(--nextui-xxx))`

File: src/components/Map/IpcMap/IpcChoropleth.tsx [hunk 3]
```diff
@@ -26,6 +28,22 @@ function IpcChoropleth({
         setSelectedCountryId={setSelectedCountryId}
         selectedCountryId={selectedCountryId}
       />
+      {!ipcRegionData && selectedCountryId && (
+        <GeoJSON
```
Comment (bohdangarchu) [inline src/components/Map/IpcMap/IpcChoropleth.tsx:32]: Maybe you could extract this into a CountrySkeleton component and pass data and color there?

**Review by bohdangarchu — CHANGES_REQUESTED** [review]
great job!!

**Review by bohdangarchu — APPROVED** [review]

**Review by marinovl7 — APPROVED** [review]
LGTM!

**Pull request merged**